src/main/java/com/microfocus/application/automation/tools/model/TestStatusModel.java

```diff
@@ -0,0 +1,33 @@
+package com.microfocus.application.automation.tools.model;
```

Comment:
If this is a new class:

1. Move it under UFT folder
2. Add license
3. Add documentation above the class explaining its purpose

Reply:
Was it removed?

Reply:
Yes it was. Not needed anymore.